packages/pwa/app/commerce-api/hooks/useWishlist.js

```diff
@@ -0,0 +1,347 @@
+                    body: {
+                        productId: id,
+                        quantity,
+                        public: false,
+                        priority: 1,
+                        type: 'product'
+                    },
+                    parameters: {
+                        customerId: customer.customerId,
+                        listId
+                    }
+                })
+
+                if (isError(response)) {
+                    throw new Error(response)
+                }
+
+                return response
+            },
+
+            /**
+             * Update an item in a customer product list
+             *
+             * @param {string} listId id of the list to update the item in
+             * @param {object} item
+             * @param {string} item.id the id of the item in the product list
+             * @param {number} item.quantity the quantity of the item
+             */
+            async _updateListItem(listId, item) {
+                const {id, quantity} = item
+                const response = await api.shopperCustomers.updateCustomerProductListItem({
+                    body: {
+                        id,
+                        quantity,
+                        public: false,
+                        priority: 1
+                    },
+                    parameters: {
+                        customerId: customer.customerId,
+                        listId: listId,
+                        itemId: item.id
+                    }
+                })
+
+                if (isError(response)) {
+                    throw new Error(response)
+                }
+
+                return response
+            },
+
+            /**
+             * Remove an item from a customer product list
+             *
+             * @param {string} listId id of the list to update the item in
+             * @param {string} itemId the id of the item in the product list
+             */
+            async _removeListItem(listId, itemId) {
+                const response = await api.shopperCustomers.deleteCustomerProductListItem(
+                    {
+                        parameters: {
+                            itemId,
+                            listId,
+                            customerId: customer.customerId
+                        }
+                    },
+                    true
+                )
+
+                if (isError(response)) {
+                    throw new Error(response)
+                }
+
+                return response
+            }
+        }
+    }, [customer.customerId, state])
+    return self
+}
+
+/**
+ * This hook is the "child class" of useCustomerProductLists.
+ * It provides functionalities to manage a single wishlist for shoppers.
+ */
+export default function useWishlist() {
+    const _super = useCustomerProductLists()
+    const {actions} = useContext(CustomerProductListsContext)
+    const PWA_DEFAULT_WISHLIST_NAME = 'PWA wishlist'
```

Comment:
i removed the one from `constant.js` because this hook should be the only one who use the constant, for separation of concerns...

(if you need to import this constant from anywhere in the app, you are probably doing it wrong.)